src/v/kafka/server/group.cc

```diff
@@ -2982,16 +2919,83 @@ ss::future<cluster::abort_group_tx_reply> group::do_abort(
     co_return make_abort_tx_reply(cluster::tx::errc::none);
 }
 
-ss::future<cluster::commit_group_tx_reply>
-group::do_commit(kafka::group_id group_id, model::producer_identity pid) {
+ss::future<cluster::commit_group_tx_reply> group::do_commit(
+  kafka::group_id group_id,
+  model::producer_identity pid,
+  model::tx_seq sequence) {
+    vlog(
+      _ctx_txlog.trace,
+      "processing do_commit_tx request: group: {}, pid: {}",
```

Comment:
i think the `group` is redundant here as the `ctx_log` prepends log entries with group id,  state and generation.